fakemenot/__init__.py

```diff
@@ -92,11 +92,18 @@ def _do_ocr_and_lookup(img_obj):
                     removed_elements += 1
                     ltweet.remove(ele)
             removal_rate = (removed_elements/float(orig_len))*100
-            if removal_rate > 75.0 :
-                print colored("[*] It looks like this is a valid tweet", 'green')
-                print colored("-> Confidence : " + "%.2f"%removal_rate + "%", 'yellow')
-                print colored("-> Potential URL : https://twitter.com/"+potential_user[1:]+"/status/"+str(tweet[1]), 'blue')
-                break
+            if removal_rate in range(75, 100):
+                print(colored("[*] It looks like this is a valid tweet", 'green'))
+                print(colored("-> Confidence : " + "%.2f"%removal_rate + "%", 'green'))
+                print(colored("-> Potential URL : https://twitter.com/"+potential_user[1:]+"/status/"+str(tweet[1]), 'green'))
+            elif removal_rate in (50, 75):
+                print(colored("[*] This might be a valid tweet", 'yellow'))
+                print(colored("-> Confidence : " + "%.2f"%removal_rate + "%", 'yellow'))
+                print(colored("-> Potential URL : https://twitter.com/"+potential_user[1:]+"/status/"+str(tweet[1]), 'yellow'))
+            elif removal_rate  < 50.0:
```

Comment:
Hey, great idea. Lets skip <50 so that it doesn't spam the terminal a lot. Thanks for the PR :)

Just pass if rate is less than 50